src/App.tsx

```diff
@@ -12,18 +12,13 @@ export const App: React.FC<{}> = () => (
   <StylesProvider injectFirst>
     <MuiThemeProvider theme={theme}>
       <ThemeProvider theme={theme}>
-        <Router>
-          <Switch>
-            <Route exact path="/">
-              <HomePage />
-            </Route>
-            {ITProfessions.map(profession => (
-              <Route path={`/details/${profession.route}`} key={profession.id}>
-                <DetailPage profession={profession} />
-              </Route>
-            ))}
-          </Switch>
-        </Router>
+        <Switch>
+          <Route exact path="/">
+            <HomePage />
+          </Route>
+          <Redirect exact path="/details" to={`/details/${ITProfessions[0].route}`} />
+          <Route path="/details/:professionId" component={DetailPage} />
```

Comment:
As Olga rightly noticed, we can't have `exact` here! :) Why?

We have a `details/:professionId` route which renders `DetailsPage`. `DetailsPage`, in turn, has a child component (`TabPanel`) that will render only if url matches nested route: `details/:professionId/categoryId`, also specified in `DetailsPage`. 

But! If we are on nested route `details/:professionId/categoryId` that means that `DetailsPage` route will not be matched if we use `exact` (`details/:professionId/categoryId` !== `details/:professionId`). So `DetailsPage` never renders which means that our nested route definition doesn't render either. Effectively, our nested route **doesn't exist** 🤷